### DISCUSSION TITLE
Setting a receipt as unshipped when it was marked as shipped

### DISCUSSION THREAD

**Discussion by NeroAzure** [etsy/open-api#980] [Q&A] [answered]
We have a use case that we want to set an receipt marked as shipped to unshipped. 
The reason for this is that a shop owner can buy shipping labels through us and we are automatically setting the tracking information in their receipts, which marks them automatically as shipped. But when they refund that label, we want to be able to mark the receipt as unshipped again.

Most of the time (about 99%) we get an error indicating that receipts marked as shipped cannot be marked as unshipped.
But <1%  seem to get through without error, I can only presume that those get sent before the first change gets applied.

In the end, the question is whether it should be possible to set this back to false or not.

**Comment by Gareth064** [comment]
In Etsy terms, "Shipped" = Fulfilled or Completed. I don't believe they allow you to uncompleted an order, not even through the website. You can only Refund completed orders.

When doing things via the API, I believe there is an "layer" between the API and the core systems which could cause a delay on things actually being updated which might be why you see <1% of your requests to "unship" an order is reported as successful.

To get a firm answer, you would be better off email Etsy Developers asking them about this. developer@etsy.com And if you get an answer, then report back here so the community can learn 👍🏻

**Reply by NeroAzure** [reply]
Thank you for your fast reply! 
I will contact them and follow up on here!

**Reply by NeroAzure** [reply, marked as answer]
A member of the Etsy team confirmed that it is indeed not possible to set an order, that has been marked as shipped, back to being unshipped.

**Reply by Gareth064** [reply]
Thanks for reporting back @NeroAzure . Others in the community will be grateful that they will find an answer to similar questions they have.
